### ISSUE TITLE
RunImageFunction ERROR [FGCP/7.0/DualAZ]

### ISSUE THREAD

**Issue by laptom** [fortinet/aws-cloudformation-templates#16]
Getting such error when running CF, 7.0: 

 RunImageFunction ERROR

{
"Status": "FAILED",
"Reason": "See the details in CloudWatch Log Stream: 2022/03/16/[$LATEST]786ee947e4d84923935a99224406f956",
"PhysicalResourceId": "2022/03/16/[$LATEST]786ee947e4d84923935a99224406f956",
"StackId": "arn:aws:cloudformation:eu-central-1:729267244622:stack/FGStack/73419f90-a550-11ec-87d7-0275f7a88d18",
"RequestId": "d002c7c1-86f5-41c3-91e9-67d47c15f483",
"LogicalResourceId": "RunImageFunction",
"NoEcho": false,
"Data": {
"msg": "error"
}
}

PLEASE CHECK QUICKLy, THX

**Comment by SgtMike70** [comment]
appears to be related to the S3 bucket that is created

**Comment by laptom** [comment]
Nope. I manually created S3 bucket. Also in the bucket I noticed two files created fgt1.txt and fgt2.txt with below content.

<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>0VBWGW849PAKT187</RequestId>
<HostId>V045eaeo0ywjPiQkhdYDhJIei2mOdeHsevq7CghnYtSjQevQs6TGvNFi82uKbePolS0BuQof/i0=</HostId>
</Error>

**Comment by SgtMike70** [comment]
Ok, be sure to check the permissions on the bucket and see that is not
blocking public access.

**Comment by laptom** [comment]
My S3 policy allows access and also I see those two emtpy files in my S3 bucket:

Any idea what might br wrong?

fgt1.txt
fgt2.tx

S3 policy:
{
  "Id": "Policy1647621792464",
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "Stmt1647621791412",
      "Action": [
        "s3:PutObject"
      ],
      "Effect": "Allow",
      "Resource": "arn:aws:s3:::tls3/*",
      "Principal": "*"
    }
  ]
}

**Comment by SgtMike70** [comment]
Each file (fgt1.txt and fgt2.txt) should have a base configuration for the
corresponding firewall.
are you using the exact name of the S3 bucket during the deployment set up
or the ARN of the bucket?
